src/ray/core_worker/lib/java/jni_init.cc

```diff
@@ -0,0 +1,75 @@
+#include "ray/core_worker/lib/java/jni_helper.h"
+
+jclass java_boolean_class;
+jmethodID java_boolean_init;
+
+jclass java_list_class;
+jmethodID java_list_size;
+jmethodID java_list_get;
+jmethodID java_list_add;
+
+jclass java_array_list_class;
+jmethodID java_array_list_init;
+jmethodID java_array_list_init_with_capacity;
+
+jclass java_ray_exception_class;
+
+jclass java_native_ray_object_class;
+jmethodID java_native_ray_object_init;
+jfieldID java_native_ray_object_data;
+jfieldID java_native_ray_object_metadata;
+
+jint JNI_VERSION = JNI_VERSION_1_8;
+
+#define LOAD_CLASS(variable_name, class_name)                     \
+  {                                                               \
+    jclass tempLocalClassRef;                                     \
+    tempLocalClassRef = env->FindClass(class_name);               \
+    variable_name = (jclass)env->NewGlobalRef(tempLocalClassRef); \
+    env->DeleteLocalRef(tempLocalClassRef);                       \
+  }
```

Comment:
It seems that we can use an inline function to instead this macro.
Does it have any issues to use inline function here?

Reply:
Good point. Updated.